src/package.json

```diff
@@ -1,13 +1,12 @@
 {
     "name": "kaholo-plugin-gitlab-trigger",
-    "version": "1.0.0",
+    "version": "1.0.1",
     "description": "",
     "main": "app.js",
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1"
     },
-    "author": "Productionmap Team (Roy Talyosef)",
+    "author": "Kaholo Team (Ilan Yaniv)",
     "license": "ISC",
-    "dependencies": {
-    }
-}
\ No newline at end of file
+    "dependencies": {}
```

Comment:
@ilanyaniv you are missing the minimatch dependency. please run `npm i -S minimatch`